src/pages/page/page.jsx

```diff
@@ -145,7 +145,7 @@ var Page = React.createClass({
    */
   generatePagesContainer: function() {
     var innerStyle = {
-      backgroundColor: this.state.styles.backgroundColor
+      backgroundColor: this.state.styles.backgroundColor || "#F2F6FC"
```

Comment:
Existing pages in edit mode without an explicit background color were showing up with no page frame, so this fixes that. Can probably be fixed elsewhere by pulling in default props, but I didn't want to go down that rabbit hole right now.